examples/IRMQTTServer/IRMQTTServer.h

```diff
@@ -4,7 +4,11 @@
  */
 #ifndef EXAMPLES_IRMQTTSERVER_IRMQTTSERVER_H_
 #define EXAMPLES_IRMQTTSERVER_IRMQTTSERVER_H_
-
```

Comment:
Nitpick: Please keep this blank line, for style/readability reasons.

Reply:
Already edited. :+1: